blocks/hero/hero.js

```diff
@@ -0,0 +1,73 @@
+import { toCamelCase } from '../../scripts/aem.js';
+import { div } from '../../scripts/dom-helpers.js';
+
+function readHeroBlockConfig(block) {
+  const config = {};
+
+  block.querySelectorAll(':scope > div').forEach((row) => {
+    if (row.children) {
+      const cols = [...row.children];
```

Comment:
do we need to shadow copy this?

Reply:
This function is just a more enhanced version of https://github.com/adobe/aem-boilerplate/blob/main/scripts/aem.js#L175-L179 . I kept what was working before and only added what I needed enhanced.